### DISCUSSION TITLE
Poetry is complaining about Fastapi version

### DISCUSSION THREAD

**Discussion by ozgurkalan** [zauberzeug/nicegui#2755] [Q&A] [answered]
### Question

Hi,
I tried adding nicegui via poetry and received  below:
```bash
(API) api (main)$ poetry add nicegui 
Using version ^1.4.19 for nicegui

Updating dependencies
Resolving dependencies... (0.1s)

Because no versions of nicegui match >1.4.19,<2.0.0
 and nicegui (1.4.19) depends on fastapi (>=0.109.1,<0.110.0), nicegui (>=1.4.19,<2.0.0) requires fastapi (>=0.109.1,<0.110.0).
So, because app depends on both fastapi (^0.110.0) and nicegui (^1.4.19), version solving failed.
````
Shall I wait for an update?

**Comment by rodja** [comment, marked as answer]
Updating to FastAPI 0.110.0 will introduce a breaking change (see https://github.com/tiangolo/fastapi/releases/tag/0.110.0). So we will probably only upgrade in upcoming 2.0 version of NiceGUI (still some time in the future). Can you do a `poetry add fastapi==0.109` before installing NiceGUI?

**Reply by ozgurkalan** [reply]
Thank you. I did and it works fine...
```bash
[tool.poetry.group.fastapi.dependencies]
fastapi = "<0.110.0"

[tool.poetry.group.ui.dependencies]
nicegui = "^1.4.19"
````
